packages/SwingSet/tools/run-utils.js

```diff
@@ -103,50 +118,26 @@ export const makeRunUtils = (controller, harness) => {
   // promise that can remain pending indefinitely, possibly to be settled by a
   // future message delivery.
 
+  const makeMethodsProxy = (invoker, target, voidResult = false) =>
```

Comment:
I would have liked some JSDoc for these arguments. In particular documenting what the signature of invoker should be

Reply:
This was actually addressed as part of #10943: https://github.com/Agoric/agoric-sdk/commit/ae4dc36c164bae2f470043af5e6671a678d354fd